### DISCUSSION TITLE
How to create a partitioned database when using couch_peruser option?

### DISCUSSION THREAD

**Discussion by sembel** [apache/couchdb#3457] [Q&A] [answered]
After creating a new user, CouchDB creates peruser-created database. But there is no option to set if the database should be partitioned.

Is it OK to delete and create a new database for the user with partitioned=true option?

**Comment by skeyby** [comment, marked as answer]
Yes, the automatic creation is just a trigger when a new org.couchdb.user is created, you can do whatever you want with the db later.

Just remember to apply the correct permission if you drop it and recreate it.

My personal suggestion, from experience, is to leave the automatic creation off (you may also find different reports about creations getting skipped in various circumstances)  and have your application create the user db on it's own when it's needed.
